src/testing/README.md

```diff
@@ -24,18 +24,18 @@ Azure Mission-Critical integrates [Azure Chaos Studio](https://aka.ms/chaosstudi
 
 The Azure Mission-Critical online reference implementation uses existing testing capabilities and frameworks whenever possible. The subsequent sections contain an overview of the used tools and frameworks.
 
-- [JMeter with Azure Load Test](#jmeter-with-azure-load-test)
-- [Locust](#locust) for load testing
+- [Azure Load Test Service](#azure-load-test-service) for load testing using the Microsoft Azure Load Test service
+- [Locust](#locust) for load testing using the open source load testing framework locust
 - [Playwright](#playwright) for UI testing
 - [Azure Chaos Studio](#azure-chaos-studio) for failure injection testing
 
-### JMeter with Azure Load Test
+### Azure Load Test Service
 
-[Azure Load Test](https://docs.microsoft.com/azure/load-testing/overview-what-is-azure-load-testing) is a managed service to execute [Apache JMeter](https://jmeter.apache.org/) load test definitions.
+Microsoft [Azure Load Test](https://docs.microsoft.com/azure/load-testing/overview-what-is-azure-load-testing) Service (MALT) is a managed service to execute [Apache JMeter](https://jmeter.apache.org/) load test definitions. Azure Mission-Critical comes with a load test definition for its sample application that can be leveraged either standalone (infrastructure and test service is deployed via a separate pipeline) or embedded into the e2e pipeline including a baseline definition.
```

Comment:
I think MALTs plan is still to add more frameworks later. So maybe rewrite to say "currently its supporting JMeter, more to follow" etc?

Reply:
We can do that. For now its only JMeter and all we do is build around that. Not sure if/when we'll see other frameworks and if that's a top priority for them.